Dockerfile

```diff
@@ -1,7 +1,6 @@
-FROM debian:jessie-slim
-MAINTAINER Rafael Römhild <rafael@roemhild.de>
+FROM phusion/baseimage:0.9.22
 
-ENV EJABBERD_BRANCH=17.08 \
+ENV EJABBERD_BRANCH=master \
```

Comment:
@balusarakesh @mpope9 I think we want the development branch here, since this docker container will be used in the context of testing locally.

Reply:
This is the third-party ejabberd which we download and compile for our usecase. It's not ideal to use development branch from other repos.

Reply:
please refer to https://github.com/skillz/ejabberd/blob/b9110370c789981272cb18ca3123dc44a1518d43/Dockerfile#L64

Reply:
Oh then good thing I asked this question, we do not want to compile the 3rd party ejabberd.
Instead, we want to compile THIS version, on the development branch.

So instead of cloning here: https://github.com/skillz/ejabberd/blob/b9110370c789981272cb18ca3123dc44a1518d43/Dockerfile#L64
This dockerfile should have access to this repository ( github.com/skillz/ejabberd ) and `make` that